### ISSUE TITLE
feat(depot): handle transportation as depot

### ISSUE THREAD

**Issue by mbayopanda** [third-culture-software/bhima#2660]
This issue is about to handle in a good way transportation, it means the situation where stock are store temporary by the transporter (truck, train, plane, ship).

The proposal is to add a flag `is_transit` (or other good name) in the depot so that we can refer to this transit depot when stock are stored at the transporter.

The complete this issue we strongly need advice from logistics team.

**Comment by jniles** [comment]
Note that we may also need specialized documents to accompany the transport, such as a [packing list](https://www.thebalance.com/import-and-export-packing-list-what-it-is-1953510).  This may prompt us to implement a more specific module to get that information.

**Comment by jeremielodi** [comment]
@jniles, what do you realy mean by packing list. are there other important properties we will need for a particular depot(transporter)?

**Comment by jniles** [comment]
Good question.

A packing list is a list of articles in a shipment.  For example, IMA ships lots of supplies to Congo, each container has a packing list that tells the warehouse what they should find in that container.

It isn't really tied to a transporter or depot, it is more tied to the process of shipping ... something we don't really support currently.  We just have stock leave and enter depots.

If you would like to propose a solution to the shipping problem, feel free 👍

**Comment by jeremielodi** [comment]
I don't think we have to propose things, @mbayopanda says this issue come from the logistic team, we have to collect every informations from them.

**Comment by jniles** [comment]
@mbayopanda did IMA ever come up with a solution to this issue?  I know that IMA did end up using some shipments, but I cannot recall how it was implemented.
